pom.xml

```diff
@@ -111,7 +111,8 @@
         <dependency.jakarta-ee-activation.version>2.0.1</dependency.jakarta-ee-activation.version>
         <dependency.jakarta-ee-jms.version>3.0.0</dependency.jakarta-ee-jms.version>
         <dependency.java-ee-activation.version>1.2.0</dependency.java-ee-activation.version>
-        <dependency.jakarta-json-api.version>1.1.6</dependency.jakarta-json-api.version>
+        <dependency.jakarta-json-api.version>2.1.2</dependency.jakarta-json-api.version>
```

Comment:
Consider using `jakarta-ee-json-api.version` and just `jakarta-ee-json-impl.version`